vnext/Shared/FeatureGate.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
```

Comment:
Because, the header is consumed by shared code, i.e. `Shared\WinRTWebSocketResource.cpp`.
Keeping the header in the Desktop folder would add a dependency from the Shared sources to the Desktop-specific ones.